xil_carma_cloud/cdasim_config/carma/lci_strategic_plugin/parameters.yaml

```diff
@@ -0,0 +1,50 @@
+# Double: multiplier to apply to the maximum allowable vehicle deceleration limit so we plan under our capabilities
+vehicle_decel_limit_multiplier : 1.0
+
+# Double: multiplier to apply to the maximum allowable vehicle acceleration limit so we plan under our capabilities
+vehicle_accel_limit_multiplier : 1.0
+
+# Double: The minimum distance in meters that the vehicle can be at before requiring a transition to the APPROACH state
+min_approach_distance : 20.0
+
+# Double: Downtrack distance until nearest intersection where the Trajectory Smoothing algorithm should activate
+trajectory_smoothing_activation_distance: 40.0
```

Comment:
this allows the lci to only start planning around workzone area. Higher distance than this may make the vehicle slow down early due to trajectory smoothing algorithm. Reducing from this too much is hard to tune for vehicle to stop at the red light.

Reply:
I just realized now the reason why reducing this was hard to tune for stopping is because inlane's is taking over the workzone area if this is reduced. When inlane is in charge, vehicle was not slowing down as fast as lci tactical plugin did and since it is too high speed before red light, it is not able to stop. This is because, I just realized, the inlane's max_accel multiplier is 0.4 which is reducing its capability to slow down. Making it 1.0, the vehicle is able to slow down dramatically faster.